topKFrequentElements.js

```diff
@@ -0,0 +1,30 @@
+// This file contains solution to the following problem:
+// https://leetcode.com/problems/top-k-frequent-elements/
+
+const topKFrequent = (nums, k) => {
+    const numsOccurences = {};
+    const occurencesNums = {};
```

Comment:
These 2 names are pretty bad (when they are together), which is which ? Consider renaming or at least leaving a clarifying comment about what is what.